source/_docs/tools/dev-tools.markdown

```diff
@@ -24,3 +24,79 @@ Screenshot of Home Assistant's Developer Tools.
 | Templates | <img src='/images/screenshots/developer-tool-templates-icon.png' alt='service developer tool icon' class="no-shadow" height="38" /> | Renders templates |
 | Info | <img src='/images/screenshots/developer-tool-about-icon.png' alt='service developer tool icon' class="no-shadow" height="38" /> | Details about Home Assistant |
 
+## What can I do with Developer Tools?
+The Developer Tools is meant for **all** (not just for the developers) to quickly try out things - like calling services, update states, raising events, and publish messages in mqtt…etc.). It is also a necessary tool for those who write custom automations and scripts by hand. The following describes each of the section in detail.
+
+{% linkable_title Services %}
+
+This section is used to call Services that are available in the ServiceRegistry.
+
+The list of services in the “Service” drop down are automatically populated based on the components that are found in the configuration, automation and script files.  If a desired service does not exist, it means either the component is not configured properly or not defined in the configuration, automation or script files.
```

Comment:
We can simplify this

The list of services in the “Service” drop down are automatically populated based on the components that you have added to Home Assistant.

Reply:
The documentation guidelines says we should not use the words like "you" when writing.  We should change that to something like "The list of services in the “Service” drop down are automatically populated based on the components that are added to Home Assistant.". 

I prefer the statement I had earlier - even though it is a bit longer, it actually makes sense.

Reply:
- `drop down` -> `dropdown`
- `a desired` -> `the desired`